diff --git a/features/ls.feature b/features/ls.feature
new file mode 100644
index 0000000..bb6d1d3
--- /dev/null
+++ b/features/ls.feature
@@ -0,0 +1,16 @@
+# features/ls.feature
+Feature: ls
+  In order to see the directory structure
+  As a UNIX user
+  I need to be able to list the current directory's contents
+
+Scenario: List 2 files in a directory
+  Given I am in a directory "test"
+  And I have a file named "foo"
+  And I have a file named "bar"
+  When I run "ls"
+  Then I should get:
+    """
+    bar
+    foo
+    """
\ No newline at end of file
* Unmerged path test/app/controllers/PostsController.php
* Unmerged path test/app/routes.php
diff --git a/test/app/tests/PostsTest.php b/test/app/tests/PostsTest.php
new file mode 100644
index 0000000..0ab38b6
--- /dev/null
+++ b/test/app/tests/PostsTest.php
@@ -0,0 +1,14 @@
+<?php
+class PostsTest extends TestCase {
+
+public function testThatTrueIsTrue()
+{
+  $this->assertTrue(true);
+}
+
+	
+}
+
+
+
+?>
\ No newline at end of file
* Unmerged path test/app/views/layout.blade.php
* Unmerged path test/app/views/posts/view.blade.php
diff --git a/test/composer.json b/test/composer.json
index 4f00769..5232e36 100644
--- a/test/composer.json
+++ b/test/composer.json
@@ -4,8 +4,12 @@
 	"keywords": ["framework", "laravel"],
 	"license": "MIT",
 	"require": {
-		"laravel/framework": "4.2.*"
+		"laravel/framework": "4.2.*",
+		"behat/behat": "2.4.*@stable"
 	},
+	"require-dev": {
+    "phpunit/phpunit": "3.7.*"
+  	},
 	"autoload": {
 		"classmap": [
 			"app/commands",
